src/pmdas/bpf/bpf.c

```diff
@@ -61,6 +61,8 @@ static pmdaOptions opts = {
     .long_options = longopts,
 };
 
+dict *pmda_config = NULL;
```

Comment:
No need to initialize globals to 0 or NULL, the compiler mandates this.

Reply:
Updated.